The version script has two problems:
1) switch is --version-script, not -version-script
2) The symbol map is missing tons of symbols (found by sun linker)

--- va/Makefile.am.orig	2021-03-23 10:54:35 UTC
+++ va/Makefile.am
@@ -76,7 +76,7 @@ libva_source_h_priv = \
 
 libva_ldflags = \
 	$(LDADD) -no-undefined	\
-	-Wl,-version-script,${srcdir}/libva.syms \
+	\
 	$(NULL)
 
 libva_cflags = \
